lib/byron/test/integration/Test/Integration/Byron/Scenario/API/Transactions.hs

```diff
@@ -690,6 +696,71 @@ scenario_TRANS_UTXO_01 fixtureSource fixtureTarget = it title $ \ctx -> do
   where
     title = "TRANS_UTXO_01 - one recipient multiple txs received"
 
+scenario_TRANS_REG_1670
+    :: forall (n :: NetworkDiscriminant) t.
+        ( DecodeAddress n
+        , EncodeAddress n
+        , PaymentAddress n IcarusKey
+        )
+    => (Context t -> [Natural] -> IO ApiByronWallet)
+    -> SpecWith (Context t)
+scenario_TRANS_REG_1670 fixture = it title $ \ctx -> do
```

Comment:
This fails before the fix.

Reply:
Since the tx input resolving is moved into the DBLayer, this test could probably be more easily done at the unit level.